packages/python/vc_init.py

```diff
@@ -191,23 +191,33 @@ def __init__(self, scope):
                 self.state = ASGICycleState.REQUEST
                 self.app_queue = None
                 self.response = {}
+                self._legacy_asyncio = sys.version_info < (3, 10)
```

Comment:
Not a fan of creating a class member variable just for checking. How about we declare a local variable at the top of the `__call__` function instead?